assets/quick-add-bulk.js

```diff
@@ -102,83 +92,37 @@ if (!customElements.get('quick-add-bulk')) {
         });
       }
 
-      updateCart(event) {
-        this.lastActiveInputId = event.target.getAttribute('data-index');
-        this.quantity.classList.add('quantity__input-disabled');
+      updateMultipleQty(items, ids) {
         this.selectProgressBar().classList.remove('hidden');
+
         const body = JSON.stringify({
-          quantity: event.target.value,
-          id: event.target.getAttribute('data-index'),
+          updates: items,
           sections: this.getSectionsToRender().map((section) => section.section),
           sections_url: this.getSectionsUrl(),
         });
 
-        fetch(`${routes.cart_change_url}`, { ...fetchConfig('javascript'), ...{ body } })
+        fetch(`${routes.cart_update_url}`, { ...fetchConfig(), ...{ body } })
           .then((response) => {
             return response.text();
           })
           .then((state) => {
             const parsedState = JSON.parse(state);
-            this.quantity.classList.remove('quantity__input-disabled');
-            if (parsedState.description || parsedState.errors) {
-              event.target.setCustomValidity(parsedState.description);
-              event.target.reportValidity();
-              this.resetQuantityInput(event.target.id);
-              this.selectProgressBar().classList.add('hidden');
-              event.target.select();
-              this.cleanErrorMessageOnType(event);
-              return;
-            }
-
-            this.renderSections(parsedState);
-
+            this.renderSections(parsedState, ids);
             publish(PUB_SUB_EVENTS.cartUpdate, { source: 'quick-add', cartData: parsedState });
+          }).catch(() => {
+            // e.target.setCustomValidity(error);
```

Comment:
This is commented out for now given that there is clamping happening + no errors returned from the BE. More info here: https://github.com/Shopify/shopify/issues/440605